tokio/src/macros/mod.rs

```diff
@@ -23,9 +23,11 @@ cfg_trace! {
     mod trace;
 }
 
-#[macro_use]
-#[cfg(feature = "rt")]
-pub(crate) mod scoped_tls;
+cfg_rt_multi_thread! {
+    #[macro_use]
+    #[cfg(feature = "rt")]
```

Comment:
does this still need the feature flag on it now that it's behind `cfg_rt_multi_thread!`?
```suggestion
```

Reply:
That block was removed entirely